Backend/src/routes/api.js

```diff
@@ -59,10 +210,20 @@ function openSocket() {
 
   // Data received listener
   client.on("data", (data) => {
-    // console.log(data);
-    console.time("update data");
-    unpackData(data);
-    console.timeEnd("update data");
+    if(data.length === bytesPerPacket) {
+      // console.log(data);
```

Comment:
Commented out code, not sure if this is necessary.

Reply:
Addressed